From 2a85700c07eb50dd4db07e19d1f7c8b76d261134 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Sun, 10 Jun 2018 10:03:38 +0200
Subject: [PATCH 170/249] wiki: Remove required_bootloader from zero devices

* This is not enforced in release tools
  therefore there's really small chance that
  anyone actually uses this bootloader version
  and results in confusing people that think
  that they really do need this specific version.

Change-Id: Ifee83b9b941fae14353ef439b545136aa2f0c252
---
 _data/devices/zerofltexx.yml | 1 -
 _data/devices/zeroltexx.yml  | 1 -
 2 files changed, 2 deletions(-)

diff --git a/_data/devices/zerofltexx.yml b/_data/devices/zerofltexx.yml
index 55285f3..c95082b 100644
--- a/_data/devices/zerofltexx.yml
+++ b/_data/devices/zerofltexx.yml
@@ -32,7 +32,6 @@ ram: 3 GB
 recovery_boot: With the device powered off, hold <kbd>Volume Up</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
   When the blue text appears, release the buttons.
 release: 2015-04
-required_bootloader: [G920FXXU5EQI8]
 screen: 129.5 mm (5.1 in)
 screen_ppi: 577
 screen_res: 2560x1440
diff --git a/_data/devices/zeroltexx.yml b/_data/devices/zeroltexx.yml
index 234fd74..d4db343 100644
--- a/_data/devices/zeroltexx.yml
+++ b/_data/devices/zeroltexx.yml
@@ -32,7 +32,6 @@ ram: 3 GB
 recovery_boot: With the device powered off, hold <kbd>Volume Up</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
   When the blue text appears, release the buttons.
 release: 2015-04
-required_bootloader: [G925FXXU5EQI8]
 screen: 130 mm (5.1 in)
 screen_ppi: 577
 screen_res: 2560x1440
-- 
2.17.1

